account/Cargo.toml

```diff
@@ -1,10 +1,7 @@
 [package]
 name = "soroban-account-contract"
 version = "0.0.0"
-authors = ["Stellar Development Foundation <info@stellar.org>"]
-license = "Apache-2.0"
```

Comment:
The examples are authored by SDF and licensed as Apache 2 so I think we should probably keep them specified as such? Or if we drop the author at least keep the license. I guess we have the license in the root so it doesn't really matter what's in the toml. 

Lgtm 👍

Reply:
I vote toss. If we're going to use these as templates we should simplify the tomls to help downstream folks not accidentally be adding authors and licenses they don't intend to